.gitignore

```diff
@@ -27,3 +27,4 @@ venv/
 # directories for dev and prod. Resource directories for prod are generated
 # via build.py and should not be committed to the repo.
 build/*
+compiled_js/
```

Comment:
Is this expected?

Reply:
Hmm if I don't have this in .gitignore, I will have to push everything in compiled_js or it will say it's dirty state. Have you ever seen this kind of error?

Reply:
Hi @sophiewu6, there was a recent note about this at #6638!

Reply:
Hmm I had no idea it was generated automatically. I removed it just now and it's working fine. Removed from .gitignore.

Reply:
You're welcome! Happy to help :)